paper/MocoPaperAppendix.tex

```diff
@@ -475,9 +475,7 @@ \subsection*{Moco's optimal control problem with prescribed kinematics}
 A common task in musculoskeletal biomechanics is to estimate the muscle and actuator behavior that drove an observed motion. We can solve this problem by minimizing the error between the observed motion and the simulated motion, as with Computed Muscle Control (using the ``slow target'')~\cite{Thelen:2003bba} or \textit{MocoTrack}. Alternatively, we can prescribe the motion exactly, as with Static Optimization~\cite{Crowninshield:1981}, electromyography-driven simulation~\cite{Lloyd:2003}, and the Muscle Redundancy Solver~\cite{Groote:2016dq}. Consider a two-dimensional point mass with coordinates $q_x$ and $q_y$ for which we prescribe a circular motion via the functions $\hat{q}_x(t) = \cos(t)$ and $\hat{q}_y(t) = \sin(t)$. We can either add these functions to the kinematic constraints $\phi(q)$, or we can substitute these functions into the equations of motion, thereby eliminating the variables $q_x$ and $q_y$. With Moco, users can choose either the former approach through OpenSim's \textit{Coordinate}, or the latter (and usually preferable) approach using \textit{PositionMotion}, a new component that employs Simbody's \textit{Motion} class. Prescribing kinematics by eliminating variables leads to a problem that is robust and fast---the nonlinear multibody dynamics are removed from the optimization problem---but prevents predicting kinematic deviations from the observed motion.
 
 When we prescribe kinematics in Moco by eliminating variables, we replace the problem in Eq~\ref{ocp} with the following:
-
 \prescribed
-
```

Comment:
The indentations occurred because an empty line indicates a new paragraph. Including these newlines was a mistake.